rewrite-java-lombok/src/main/java/org/openrewrite/java/lombok/LombokSupport.java

```diff
@@ -59,8 +59,13 @@ public class LombokSupport {
         }
 
         String oldValue = System.setProperty("shadow.override.lombok", String.join(File.pathSeparator, overrideClasspath));
+        Class<?> shadowLoaderClass;
+        try {
+            shadowLoaderClass = Class.forName("lombok.launch.ShadowClassLoader", true, parserClassLoader);
+        } catch (Exception e) {
```

Comment:
I think it's better to only catch:

```suggestion
        } catch (ClassNotFoundException e) {
```

Reply:
Invoking `Class.forName` can throw different kinds of exceptions, In this case I think catching all is okay as in any of these cases I think we want to retry